docs/README.md

```diff
@@ -27,3 +27,26 @@ If the used Docker release supports multistage builds, you can enable this by se
 ```
 $ MULTISTAGE_BUILD=1 make build
 ```
+
+## Catalog build
+1. Create a new CSV file using the latest ember-csi container. Replace 0.9.x
+   with the next version you want to use.
+```
+$ mkdir -p ../../deploy/olm-catalog/0.9.x
+$ cp deploy/olm-catalog/0.9.4/ember-csi-operator.crd.yaml deploy/olm-catalog/0.9.x/ember-csi-operator.crd.yaml
+$ cd deploy/olm-catalog/
+$ podman pull embercsi/ember-csi:master
+$ podman run --rm embercsi/ember-csi:master ember-list-drivers -d | CONSOLE_VERSION=4.5 python2 ./yaml-options-gen.py > ../../deploy/olm-catalog/0.9.x/ember-csi-operator.v0.9.x.clusterserviceversion.yaml
```

Comment:
We have a script to generate the options, we should use it instead of doing it manually (L38-L39).
We should also update said script to pull the container to avoid using an old container.

Reply:
Good point, done